src/producer.rs

```diff
@@ -9,6 +9,8 @@ use cortex_m::{interrupt, register};
 #[defmt::global_logger]
 pub struct Logger;
 
+defmt::timestamp!("");
+
 impl defmt::Write for Logger {
     // fn start_of_frame(&mut self, len: usize) {
```

Comment:
Could you remove these commented out functions? 
They were just a reminder of the multi-write issue, which you have now fixed :+1:

Reply:
Sure!

Reply:
@MathiasKoch done